### ISSUE TITLE
Vite/ESM support

### ISSUE THREAD

**Issue by okydk** [revolist/vue-datagrid#5]
Looks very promising!
However I'm getting errors while trying the vue 3 version with Vite?
```
11:13:52 AM [vite] Internal server error: Failed to resolve import ".entry.js". Does the file exist?
  Plugin: vite:import-analysis
  File: /node_modules/.vite/@revolist_vue3-datagrid.js?v=dca1123c
  1687|    return import("./" + a + ".entry.js").then(function(e2) {
     |                  ^
```

**Comment by revolist** [comment]
Hello @okydk, I have investigated your problem, and seems it's an issue from Vite.
They don't support relative imports well hence it fails for external libraries. They have tried to fix it here but fix didn't cover all.
https://github.com/vitejs/vite/commit/bbfe06ce1af8f89490032e609377c6516f7da773

**Comment by revolist** [comment]
Hello @okydk, sorry for a long answer. 
Starting from version 3.0 everything should work just fine. I have updated your sample with a working example:

https://github.com/revolist/vue3-revolist-sample
We have switched to esm modules and direct import.

But 3.0 still in beta, we are improving some things such as compatibility with modern frameworks, build systems such as vite and bootstrap support.

"@revolist/vue3-datagrid": "^3.0.0-beta.1"
